### DISCUSSION TITLE
Why this different behavior?

### DISCUSSION THREAD

**Discussion by stefanbode** [jomjol/AI-on-the-edge-device#1333] [General]
The first one is correct and fine, but why does the second one evaluates the digit to "9"? Even if the "vorgaenger" is 6.0 it is more likely that the "8.8" is a 8 and not a 9 

: ClassFlowCNNGeneral::ZeigerEvalHybridNeu - Analoger Vorgänger, Bewertung über ZeigerEvalAnalogNeu = 8 zahl: 8.704139 zahl_vorgaenger = 5.768895 eval_vorgaenger = 5 DigitalUnschaerfe = 0.200000

: ClassFlowCNNGeneral::ZeigerEvalHybridNeu - Analoger Vorgänger, Bewertung über ZeigerEvalAnalogNeu = 9 zahl: 8.854032 zahl_vorgaenger = 5.843235 eval_vorgaenger = 5 DigitalUnschaerfe = 0.200000

**Comment by jomjol** [comment]
Well, there are some counters, that stop rotation at 8.8 - it really depends on the type of your meter. Therefore currently the band around the full number is +/-0.2.

**Comment by stefanbode** [comment]
ok, a 8.85 is less than 0.2 away from 9. Therefore this is treated as nine if precessor is 5.8. What if the precessor if 8.7? in this case it is more likly than the 8.85 is what it is. Is there a config option to change the "DigitaleUnschaerfe"? The "problem" is from the new german water meter with digital interface. The last DIGIT move continuously. There is no start/stop on the last digit anymore. The other digits behave normal. Start rotating at 9 and finish latest at 1.

**Comment by jomjol** [comment]
No, this currently not parametrized.
